xep-0369.xml

```diff
@@ -204,20 +204,20 @@
       <p>Private Messages are not stored in the messages node.</p>
     </section3>
     <section3 topic="Subject Node" anchor="subject-node">
-      <p>The subject node publishes the current subject of channel.  Subject history is stored in MAM.    The id of each node is the current value of the channel subject.   A single item is stored in this node at a time.  Users subscribe to this node to receive subject updates. The following example shows the format of a item in the Subject node.</p>
-      <example caption="Subject Node"><![CDATA[
-<items node='urn:xmpp:mix:nodes:subject'>
-  <item id='How to use Toads' />
-</items>
-]]></example>
+      <p>The subject node publishes the current subject of channel. Subject history is stored in MAM. A single item is stored in this node at a time which may contain one or more "text" elements that SHOULD posess an xml:lang attribute that describes the natural language of the subject. Users subscribe to this node to receive subject updates. The following example shows the format of a item in the subject node.</p>
```

Comment:
I think a SHOULD for something that generally impractical is wrong. I'd rather MUST possess xml:lang attributes if more than one text element is present.